internal/metrics/metrics.go

```diff
@@ -0,0 +1,26 @@
+package metrics
```

Comment:
Any reason this can't use the opentelemetry go sdk?

Reply:
take a look at [531320f](https://github.com/storacha/etracker/pull/6/commits/531320f5f8bc21ae5260fc147da054e306482582) and let me know if it matches what you had in mind

Reply:
`ConsolidatedBytesPerNode` could probably be an integer counter, will leave it up to you. Looks good otherwise - thanks!